fixtures/git_content_repos.py

```diff
@@ -1,10 +1,36 @@
 import pytest
+import json
+import urllib
+import urllib.error
+import urllib.parse
+import urllib.request
+
+import requests
 
 
 @pytest.fixture
-def git_content_repos():
-    """Returns the list of all github content repos
+def git_content_repos(headers_data):
+
+    """Returns the list of all github content repos (starting with osbooks-)
     """
-    repo = ["osbooks-university-physics", "osbooks-college-algebra-bundle"]
 
-    return repo
+    url = "https://api.github.com/orgs/openstax/repos?per_page=50&type=private"
+
+    resp = requests.get(url, headers=headers_data)
+
+    repos = []
+
+    while resp.links.get("next"):
+        next_url = resp.links["next"]["url"]
+        resp = requests.get(next_url, headers=headers_data)
```

Comment:
The way this loop structure is setup, I think you may actually end up ignoring the first batch of 50 responses (it just so happens the repos you want end up on page 2, so this may not be obvious).

Reply:
@rnathuji Yeees. That is why when I added page=100, it did not give me any repos but with 25 or 50 it gave me 6 osbooks- repos. hm...